dj_cqrs/mixins.py

```diff
@@ -404,3 +408,54 @@ def cqrs_delete(cls, master_data):
             raise NotImplementedError
 
         return cls.cqrs.delete_instance(master_data)
+
+    @staticmethod
+    def should_retry_cqrs(current_retry, exception=None):
+        """Checks if we should retry the message after current attempt.
+
+        :param current_retry: Current number of message retries.
+        :type current_retry: int
+        :param exception: Exception instance raised during message consume.
+        :type exception: Exception, optional
+        :return: True if message should be retried, False otherwise.
+        :rtype: bool
+        """
+        default_max_retries = 10
+        max_retries = settings.CQRS.get('max_retries', default_max_retries)
+        if max_retries is None:
+            # Infinite
+            return True
+
+        min_value = 0
+        if not isinstance(max_retries, int) or max_retries < min_value:
+            logger.warning(
+                "Settings max_retries={} is invalid, using default {}".format(
+                    max_retries, default_max_retries,
+                )
+            )
+            max_retries = default_max_retries
+
+        return current_retry < max_retries
```

Comment:
Verify, please, that we need strict comparison here.

Reply:
max_retries = 3
| attempt        | current_retry           | should retry?  |
|-------------|-------------|-----|
| 1      | 0 | 0 < 3 = True |
| 2      | 1      |   2 < 3 = True |
| 3 | 2      |    2 < 3 = True |
| 4 | 3      |    3 < 3 = False |

total attempts = first + retries